LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2016 haya14busa
+Copyright (c) 2016 - present haya14busa
```

Comment:
In my understanding, we don't need to update copyright of year and I'm not sure `- present` is actually valid.
Can you revert the change?